ibis/expr/api.py

```diff
@@ -4032,6 +4032,72 @@ def _table_count(self):
     return ops.Count(self, None).to_expr().name('count')
 
 
+def _table_dropna(self, subset: Optional[List[str]] = None, how: str = 'any'):
+    """
+    Remove rows with null values from the table.
+
+    Parameters
+    ----------
+    subset : list of strings
+      Optional, columns names to consider. Defaults to all columns.
+    how : string
+      Determine whether a row is removed if there is at least one null
+      value in the row ('any'), or if all row values are null ('all').
+      Options are 'any' or 'all'. Default is 'any'.
+
+    Examples
+    --------
+    >>> import ibis
+    >>> t = ibis.table([('a', 'int64'), ('b', 'string')])
```

Comment:
i prefer these comments to be on a separate line *before*, but whatever is consistent other doc strings

Reply:
This was done for consistency with the other examples in this file